dyn_em/start_em.F

```diff
@@ -142,6 +145,10 @@ SUBROUTINE start_domain_em ( grid, allowed_to_read &
 #if (WRF_CHEM != 1)
          ALLOCATE(CLDFRA_OLD(IMS:IME,KMS:KME,JMS:JME),STAT=I)  ; CLDFRA_OLD = 0.
 #endif
+#if (EM_CORE == 1)
```

Comment:
@smileMchen @liujake @jjguerrette 
Ming,
We are in the start_em.F file, so the EM_CORE == 1 ifdef is not required. There are other fields that are allocated on the heap, but we should at least deallocate this field after it is used.

Reply:
@davegill 
Dave,
I have fixed this issue. Thank you!